mobile/test/swift/EngineBuilderTests.swift

```diff
@@ -421,6 +405,46 @@ final class EngineBuilderTests: XCTestCase {
     self.waitForExpectations(timeout: 0.01)
   }
 
+  func testAddingRtdsAndAdsConfigurationWhenRunningEnvoy() {
+    let bootstrapDebugDescription = EngineBuilder()
+      .addEngineType(MockEnvoyEngine.self)
+      .addRtdsLayer(name: "rtds_layer_name", timeoutSeconds: 14325)
+      .setAggregatedDiscoveryService(address: "FAKE_SWIFT_ADDRESS", port: 0)
+      .bootstrapDebugDescription()
+    print(bootstrapDebugDescription)
+    XCTAssertTrue(bootstrapDebugDescription.contains("rtds_layer_name"))
+    XCTAssertTrue(bootstrapDebugDescription.contains(#"initial_fetch_timeout { seconds: 14325 }"#))
```

Comment:
Nit: no need for a raw string literal here since nothing needs to be escaped:

```suggestion
    XCTAssertTrue(bootstrapDebugDescription.contains("initial_fetch_timeout { seconds: 14325 }"))
```